src/index.ts

```diff
@@ -31,8 +31,11 @@ program.command('test').action(async () => {
   konsola.title(`Test`, '#8556D3', 'Attempting a test...')
   const verbose = program.opts().verbose
   try {
-    // await loginWithEmailAndPassword('aw', 'passwrod', 'eu')
-    await loginWithToken('WYSYDHYASDHSYD', 'eu')
+    const { state, initializeSession } = session()
+
+    await initializeSession()
+
+    console.log(state)
```

Comment:
do we need this log here?

Reply:
This command is for quick testing purposes, is gonna be removed on the final release, consider it as a playground for now

Reply:
ok!